packages/at_client/lib/src/service/notification_service_impl.dart

```diff
@@ -26,8 +26,8 @@ class NotificationServiceImpl
   final Map<NotificationConfig, StreamController> _streamListeners =
       HashMap(equals: _compareNotificationConfig, hashCode: _generateHashCode);
   final emptyRegex = '';
-  static const notificationIdKey = '_latestNotificationIdv2';
-  static const lastReceivedNotificationKey = 'lastReceivedNotification';
+  static const notificationIdKey = '_latestnotificationidv2';
```

Comment:
is there a need to change the constant ? since we are lower casing the key anyway before storing and in processing

Reply:
It's for the tests, we are checking for this key in the keystore and since the key will now be in lowercase the tests are failing. Alternative we can let the key be the same as before and use key.toLowerCase in the expect method of tests

Reply:
imo Better to use key.toLowerCase in the expects in the tests

Reply:
I'll revert the variable name and expect key.lowercase(). Will push that change as part of the next commit

Reply:
This change has been done as part of ba098634f6a36c8e5d0b93eef775d50a3d111970